components/exploits/ExploitChecker.tsx

```diff
@@ -20,7 +20,16 @@ const ExploitChecker = ({ exploit }: Props) => {
   const { data: status } = useQuery({
     queryKey: ['exploit-status', exploit.slug, allowances?.map(getAllowanceKey)],
     queryFn: () => {
-      const status = isAffectedByExploit(allowances, exploit) ? 'affected' : 'safe';
+      //const status = isAffectedByExploit(allowances, exploit) || 'affected' || 'previously_accessible : 'safe';
+      const { affected, previouslyAccessible } = isAffectedByExploit(allowances, exploit);
+      let status;
+      if (affected) {
+        status = 'affected';
+      } else if (previouslyAccessible) {
+        status = 'previously_accessible';
+      } else {
+        status = 'safe';
+      }
```

Comment:
Let's merge this code with the `isAffectedByExploit` and change the name to `getExploitStatus` or something similar.